src/app/(Authentication)/SignUp/Address/styles.ts

```diff
@@ -0,0 +1,126 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../../../styles/colors';
+
+export default StyleSheet.create({
+  container: {
+    backgroundColor: colors.white,
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+  },
+  header: {
+    justifyContent: 'flex-start',
+    flexDirection: 'row',
+    marginTop: 90,
+    marginRight: 100,
+    marginBottom: 40,
+  },
+  image: {
+    marginTop: 2,
+  },
+  backText: {
+    color: colors.midGrey,
+    fontSize: 16,
+    fontWeight: '400',
+    marginLeft: 7,
+  },
+  backButton: {
+    marginRight: 82,
+  },
+  instructionText: {
+    color: colors.black,
+    fontSize: 24,
+    fontWeight: '800',
+    marginRight: 25,
+    marginBottom: 20,
+  },
+  inputBox: {
+    marginTop: 11,
+    width: '100%',
+    alignItems: 'center',
+  },
+  inputWrap: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  smallInputBox: {
+    marginTop: 11,
+  },
+  displayTextSmall: {
+    fontSize: 13,
+    opacity: 0.35,
+  },
+  inputSmall: {
+    height: 40,
+    borderBottomWidth: 0.75,
+    width: 147,
+    borderColor: colors.darkGrey,
+  },
+  inputSmallFocused: {
+    height: 40,
+    borderBottomWidth: 2,
+    width: 147,
+    borderColor: colors.darkRed,
+  },
+  space: {
+    margin: 11,
+  },
+
+  space2: {
+    marginBottom: 15,
+  },
+  nextText: {
+    color: colors.white,
+    fontSize: 16,
+    opacity: 1,
+    fontWeight: '600',
+    marginLeft: 85,
+    marginRight: 20,
+  },
+  nextButton: {
+    backgroundColor: colors.black,
+    marginBottom: 20,
+    borderRadius: 3,
+    borderColor: colors.black,
+    width: 310,
+    height: 50,
+    padding: 15,
+    flexDirection: 'row',
+    // marginTop: 300,
+  },
+  nextButtonGray: {
+    backgroundColor: colors.midGrey,
+    marginBottom: 20,
+    borderRadius: 3,
+    borderColor: colors.midGrey,
+    width: 310,
+    height: 50,
+    padding: 15,
+    flexDirection: 'row',
+  },
+  nextButtonDown: {
```

Comment:
Josh and I decided that we don't need to move the next button down when the keyboard isn't in focus. It only adds additional complexity without much design/accessibility benefit. So we won't need the `nextButtonDown` style option.